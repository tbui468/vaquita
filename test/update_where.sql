create database sol;
open sol;

create table planets (id int key, name string, mass float, atmosphere bool, moons int);
insert into planets (id, name, mass, atmosphere, moons) values (1, "Venus", 20.0, false, 0), (2, "Earth", 10.0, true, 1), (3, "Mars", 10.0, false, 2);
update planets set name="new_name", mass=0.0, atmosphere=true, moons=0 where id = 2;
select * from planets;

close sol;
drop database sol;
exit;
