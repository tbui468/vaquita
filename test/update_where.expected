created database sol
opened database sol
created table planets
inserted 3 record(s) into planets
1 row(s) updated
id, name, mass, atmosphere, moons
1, Venus, 20.000000, false, 0
2, new_name, 0.000000, true, 0
3, Mars, 10.000000, false, 2
closed database sol
dropped database sol
